Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -183,10 +184,16 @@ public void run() {
             adSizeType = BANNER_LEADERBOARD;
         } else if (adWidthInDp >= BANNER.getWidth() && adHeightInDp >= BANNER.getHeight()) {
             adSizeType = BANNER;
-        } else {
+        } else if (adWidthInDp >= BANNER_SHORT.getWidth() && adHeightInDp >= BANNER_SHORT.getHeight()) {
             adSizeType = BANNER_SHORT;
         }
 
+        if (adSizeType == null) {
+            Log.d(ADAPTER_NAME, "no matched ad size for requesting ad size:" + adWidthInDp + "x" + adHeightInDp);
```

Comment:
shouldn't we use MoPubLog.log(mPlacementId, CUSTOM, ADAPTER_NAME, ....?

Reply:
Good suggestion: fixed.